aiida/backends/tests/cmdline/commands/test_calcjob.py

```diff
@@ -96,14 +91,13 @@ def setUpClass(cls, *args, **kwargs):
         calc.set_process_state(ProcessState.FINISHED)
         cls.calcs.append(calc)
 
-        # Uncomment when issue 2426 is addressed
-        # # Load the fixture containing a single ArithmeticAddCalculation node
-        # import_archive_fixture('calcjob/arithmetic.add.aiida')
+        # Load the fixture containing a single ArithmeticAddCalculation node
+        import_archive_fixture('calcjob/arithmetic.add.aiida')
```

Comment:
just a question - in which sense is this a fixture? @yakutovicha was also confused by this.
In my understanding, the term `fixture` in python is mainly used by `pytest`, where it has a [very specific meaning](https://docs.pytest.org/en/latest/fixture.html).

Can we remove the `_fixture` and also move the `import_archive` simply to something like `aiida.backends.tests.utils.archive`?

Reply:
My understanding is that `fixture` means files that are present as "records", though I will have to ask @sphuber for the original meaning and reasoning behind this name.

Reply:
I used the term fixture as the archive is a resource required for the testing, which matches some description for example [found here](https://en.wikipedia.org/wiki/Test_fixture#Software). Note that test fixtures can come in various forms and especially in the software analogue there are various ways of 'delivering' or setting up the fixture. `pytest` specifically implements them as a magic way to inject them as arguments into the test function. Here the fixture is just a data archive that is imported to populate the database. I don't think it is any less valid, but I am fine with renaming it to just archive.

Reply:
I guess what I meant is that in most of the aiida-core testbase we are not using `fixture` in this more general sense (although it would apply to many other functions as well).
Using it here for the archive seems to suggest this function is "special" and can lead to confusion.
I'd vote to remove it for now.

Reply:
> Can we remove the `_fixture` and also move the `import_archive` simply to something like `aiida.backends.tests.utils.archive`?

I will do this.

Reply:
As an addition - should I rename `aiida.backends.tests.fixtures` to `aiida.backends.tests.archives` as well?

Reply:
Aren't these tests for the actual plugin fixtures?

Reply:
No. This path is the top-directory, wherein all export archives are sorted and stored to use with tests.

Reply:
I see, they aren't. Yes, good to rename!

Reply:
Okay. In this PR or a separate one, since this will probably touch a lot of non-migration-related tests?

Reply:
can be done in a separate PR

Reply:
> can be done in a separate PR

Great. Opened issue #2817 for this.